### PR TITLE
Change default connector url to include an http prefix

### LINKED ISSUE #1085: Change default connector url to include an http prefix
Depends on completion of ~~https://github.com/fybrik/fybrik/issues/1049~~

### PR DESCRIPTION
**Pull request by revit13**
This PR addresses issue #1085 : it changes the default connector url to include an http prefix

Closes #1085

### CODE REVIEW

**Review by shlomitk1 — APPROVED** [review]

File: charts/fybrik/templates/fybrik-config.yaml [hunk 1]
```diff
@@ -11,9 +11,9 @@ data:
   DATAPATH_MAX_SIZE: {{ .Values.manager.dataPathMaxSize | quote }}
   CONNECTION_TIMEOUT: {{ .Values.manager.connectionTimeout | default .Values.global.connectionTimeout | quote }}
   CATALOG_PROVIDER_NAME: {{ .Values.coordinator.catalog | quote }}
-  CATALOG_CONNECTOR_URL: {{ .Values.coordinator.catalogConnectorURL | default (printf "%s-connector:80" .Values.coordinator.catalog) | quote }}
+  CATALOG_CONNECTOR_URL: {{ .Values.coordinator.catalogConnectorURL | default (printf "http://%s-connector:80" .Values.coordinator.catalog) | quote }}
```
Comment (roee88) [inline charts/fybrik/templates/fybrik-config.yaml:14]: There is another one in `opa-connector-cm.yaml`

Comment (roee88) [reply charts/fybrik/templates/fybrik-config.yaml:14]: and probably other places where CATALOG_CONNECTOR_URL and/or MAIN_POLICY_MANAGER_CONNECTOR_URL are used in tests.

File: charts/fybrik/values.yaml [hunk 2]
```diff
@@ -66,16 +66,14 @@ coordinator:
   catalog: "katalog"
 
   # Overrides the catalog connector URL.
-  # Defaults to `<catalog>-connector:80`.
```
Comment (roee88) [inline charts/fybrik/values.yaml:69]: Keeping a comment would be useful. Like `Defaults to `http://<catalog>-connector:80`.

File: charts/fybrik/values.yaml [hunk 3]
```diff
@@ -66,16 +66,14 @@ coordinator:
...
-  catalogConnectorURL: "http://katalog-connector:80"
+  catalogConnectorURL: ""
 
   # Configures the policy manager system name to be used by the coordinator manager.
   # Accepted values are "opa" or any meaningful name if a third party connector is used.
   policyManager: "opa"
 
   # Overrides the policy manager connector URL.
-  # Defaults to `<policyManager>-connector:80`.
```
Comment (roee88) [inline charts/fybrik/values.yaml:77]: Same comment (`Defaults to `http://<policyManager>-connector:80`.)

File: charts/fybrik/templates/fybrik-config.yaml (hunk 1 above)
Comment (revit13) [reply charts/fybrik/templates/fybrik-config.yaml:14]: Thanks, I changed that

File: charts/fybrik/values.yaml (hunk 2 above)
Comment (revit13) [reply charts/fybrik/values.yaml:69]: Done. Thanks

File: connectors/opa/Makefile [hunk 4]
```diff
@@ -10,7 +10,7 @@ NEED_TEST_COVERAGE_STATISTICS=1
 NEED_TEST_COVERAGE_STATISTICS_IN_HTML=0
 
 CATALOG_PROVIDER_NAME ?= "katalog"
-CATALOG_CONNECTOR_URL ?= "katalg-connector:50090"
+CATALOG_CONNECTOR_URL ?= "http://katalg-connector:50090"
```
Comment (cdoron) [inline connectors/opa/Makefile:13]: katalg should be katalog, right?

Comment (revit13) [reply connectors/opa/Makefile:13]: @cdoron Thanks! fixing

**Review by cdoron — APPROVED** [review]

**Review by roee88 — APPROVED** [review]

**Pull request merged**